components/delegations/cells/DelegatorCell.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
+
+import AddressDisplay from 'components/address/AddressDisplay';
+import type { Delegation } from 'lib/delegate/DelegatePlatform';
+
+interface Props {
+  delegation: Delegation;
+}
+
+const DelegatorCell = ({ delegation }: Props) => {
+  return <AddressDisplay address={delegation.delegator} />;
```

Comment:
I recommend returning `AddressCell` here so that it also includes the link / copy button automatically.